LPS-130463 Implement Google Cloud translator

adolfopa opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

adolfopa commented:
@boton Here is an initial implementation of the google autotranslation service. This includes both the translation service and the latest changes to enable POST access (no changes since last pr).

Please put your changes on top of this and resend back to me for proper testing.

Thanks!

adolfopa commented:
See https://github.com/liferay-lima/liferay-portal/pull/1747
